src/ui/public/vislib/visualizations/_map.js

```diff
@@ -47,16 +51,15 @@ export default function MapFactory(Private) {
     this._valueFormatter = params.valueFormatter || _.identity;
     this._tooltipFormatter = params.tooltipFormatter || _.identity;
     this._geoJson = _.get(this._chartData, 'geoJson');
-    this._mapZoom = params.zoom || defaultMapZoom;
+    this._mapZoom =  Math.max(Math.min(params.zoom || defaultMapZoom, tilemapOptions.maxZoom), tilemapOptions.minZoom);
     this._mapCenter = params.center || defaultMapCenter;
     this._attr = params.attr || {};
 
     let mapOptions = {
-      minZoom: 1,
-      maxZoom: 18,
+      minZoom: tilemapOptions.minZoom,
+      maxZoom: tilemapOptions.maxZoom,
       noWrap: true,
       maxBounds: L.latLngBounds([-90, -220], [90, 220]),
-      scrollWheelZoom: false,
```

Comment:
I'm not sure why it is there in the first place, but this isn't the right PR in which to change the behavior, especially since that configuration is being passed in 4.x.  I'll add it back in for now.

@pickypg If you think it should be removed, feel free add an issue for it.